robottelo/ui/hosts.py

```diff
@@ -291,3 +291,9 @@ def fetch_host_parameters(self, name, domain_name, parameters_list):
             self.click(tab_locator)
             result[param_name] = self.wait_until_element(param_locator).text
         return result
+
+    def get_host_property(self, parameter_name):
```

Comment:
@oshtaier got it now, thanks. Still, why totally opposite approach? I'd like to standardize it, both methods are doing basically the same thing on different pages, but one accepts list of all properties and handles navigation, the second one only returns one property and wants user to handle navigation.